client/src/components/UtcDate.vue

```diff
@@ -22,33 +22,22 @@ export default {
         },
         customFormat: {
             type: String,
+            default: null,
```

Comment:
why?

Reply:
Props should always have explicit defaults.